util/Conventional_Monitor/CMakeLists.txt

```diff
@@ -87,13 +81,6 @@ if(CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
     include_directories(${MPI_Fortran_INCLUDE_DIRS} ${MPI_INCLUDE_PATH} "./" ${CMAKE_INCLUDE_OUTPUT_DIRECTORY})
     link_directories(${MPI_Fortran_LIBRARIES} ${ARCHIVE_OUTPUT_PATH} )
     find_package( NetCDF REQUIRED)
```

Comment:
This will only get you the C library unless the Fortran COMPONENT is requested.

Reply:
Correct.   Revised `CMakeLists.txt` contains
`   find_package( NetCDF COMPONENTS C Fortran REQUIRED)
`